lib/IRGen/IRGenModule.cpp

```diff
@@ -1216,7 +1221,7 @@ static bool isFirstObjectFileInModule(IRGenModule &IGM) {
 void IRGenModule::emitAutolinkInfo() {
   // Collect the linker options already in the module (from ClangCodeGen).
   // FIXME: This constant should be vended by LLVM somewhere.
-  auto *Metadata = Module.getOrInsertNamedMetadata("llvm.linker.options");
+  auto *Metadata = Module.getOrInsertNamedMetadata("llvm.dependent-libraries");
```

Comment:
I don't think that this change is correct - its going to break non-ELF targets.

Reply:
Ah, serendipity! Now I have more references to this `llvm.dependent-libraries` thing for #31912 🙂